fastlane/Fastfile

```diff
@@ -492,6 +513,8 @@ platform :ios do
       './Tests/InstallationTests/SPMCustomEntitlementComputationInstallation/SPMInstallation.xcodeproj/project.pbxproj',
       './Tests/InstallationTests/ReceiptParserInstallation/ReceiptParserInstallation.xcodeproj/project.pbxproj',
       './Tests/APITesters/CustomEntitlementComputationSwiftAPITester/CustomEntitlementComputationSwiftAPITester.xcodeproj/project.pbxproj',
+      # Fixme: this project is currently pointing to `paywalls` branch, it needs to be changed to `main` once that's merged.
```

Comment:
do you need to update line 527?

Reply:
Oh I already did this so let me remove the comment thanks :D

Reply:
The project is pointing to `main` so this works now.